.github/workflows/rebuild.yml

```diff
@@ -17,8 +17,17 @@ jobs:
     - name: setup
       run: ./bin/setup-github-workflow
 
-    - name: rebuild
-      run: ./bin/rebuild
+    - name: rebuild-gisiad-data
+      run: ./bin/rebuild gisaid
+      env:
+        AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
+        SLACK_CHANNELS: ncov-gisaid-updates
+
+    - name: rebuild-genbank-data
+      run: ./bin/rebuild genbank
```

Comment:
GH won't let me comment on the line directly, but the `SLACK_CHANNELS` env for this job step is still `ncov-gisaid-updates` instead of `ncov-genbank-updates`. (And really, we'll want to update it to `ncov-open-updates` in the future, but that requires some additional coordination.)

Reply:
Ahh - I noticed this but didn't realise there was a `ncov-genbank-updates` channel. Will update the PR.